includes/send_rx_functions.php

```diff
@@ -7,7 +7,8 @@
 include_once dirname(APP_PATH_DOCROOT) . '/vendor/autoload.php';
 
 use ExternalModules\ExternalModules;
-use UserProfile\UserProfile;
+    use SendRx\ExternalModule\ExternalModule;
+    use UserProfile\UserProfile;
```

Comment:
```suggestion
use UserProfile\UserProfile;
```
Don't load in the ExternalModule class since we're not using it directly